packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -55,11 +57,14 @@ export async function prefetchLatestSnapshot(
         true /* throwOnNullToken */,
     );
 
-    const snapshotDownloader = async (url: string, fetchOptions: {[index: string]: any}) => {
-        return fetchAndParseAsJSONHelper(
-            url,
-            fetchOptions,
-        );
+    const snapshotDownloader = async (
+        finalOdspResolvedUrl: IOdspResolvedUrl,
+        storageToken: string,
+        snapshotOptions: ISnapshotOptions | undefined,
+        controller?: AbortController,
+    ) => {
+        return downloadSnapshot(
+            finalOdspResolvedUrl, storageToken, snapshotOptions, fetchBinarySnapshotFormat, controller);
```

Comment:
I wonder if fetchBinarySnapshotFormat should be added to snapshotOptions somehow such that we do not need to have these kind of one-liner wrappers...

Reply:
For example, we can leave ISnapshotOptions as is but create something like this:
```
interface ISnapshotOptionsEx {
   spoOptions: ISnapshotOptions;
   V2: boolean;
}
```

Reply:
I will follow up with this suggestion in next PR.